<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xem_lai_quy</class>
 <widget class="QMainWindow" name="xem_lai_quy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xem lại các quỹ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string>CÁC QUỸ ĐÃ TẠO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuayLai">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 40, 145, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ic_arrow.png</normaloff>../ic_arrow.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblXemLai">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>440</width>
      <height>220</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="btnChinhSua">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 209);
border-radius:10px</string>
    </property>
    <property name="text">
     <string>Chỉnh sửa </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoa">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 209);
border-radius:10px</string>
    </property>
    <property name="text">
     <string>Xoá</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
